public/app/plugins/datasource/cloud-monitoring/types.ts

```diff
@@ -128,6 +129,8 @@ export interface MetricQuery extends BaseQuery {
   preprocessor?: PreprocessorType;
   // To disable the graphPeriod, it should explictly be set to 'disabled'
   graphPeriod?: 'disabled' | string;
+  title?: string;
+  text?: string;
```

Comment:
Maybe it would be a bit more explicit to put these two fields in a `AnnotationMetricQuery` interface that extends `MetricQuery`. Thoughts?